lib/middleware-common/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "wasmer-middleware-common"
+version = "0.3.0"
+repository = "https://github.com/wasmerio/wasmer"
+description = "Wasmer runtime common middlewares"
+license = "MIT"
+authors = ["The Wasmer Engineering Team <engineering@wasmer.io>"]
+edition = "2018"
+
+[dependencies]
+wasmer-runtime-core = { path = "../runtime-core", version = "0.3.0" }
```

Comment:
@losfair Should we include this in runtime core since it is small and with few dependencies?

Reply:
There will be more "utility"-like middlewares in this crate (e.g. metering). I think it would be better to keep it separate.

Reply:
```suggestion
wasmer-runtime-core = { path = "../runtime-core", version = "0.4.0" }
```